app/services/salesforce_service.rb

```diff
@@ -44,7 +44,12 @@ def self.eligible_listings(filters)
 
   # get one detailed listing result by id
   def self.listing(id)
-    api_get("/services/apexrest/Listings/#{id}").first
+    api_get("/services/apexrest/ListingDetails/#{id}").first
+  end
+
+  # get all units for a given listing
+  def self.units(listing_id)
+    api_get("/services/apexrest/Listing/Units/#{listing_id}")
```

Comment:
not really for you but why isn't vertiba being RESTful? `/listings/listing_id/units`... haha